eZ/Bundle/EzPublishCoreBundle/EventListener/ViewControllerListener.php

```diff
@@ -59,7 +59,7 @@ public function __construct(
 
     public static function getSubscribedEvents()
     {
-        return array( KernelEvents::CONTROLLER => 'getController' );
+        return array( KernelEvents::CONTROLLER => array( 'getController', 10 ) );
```

Comment:
Remember that we can now use the array short syntax. On those calls, it really reads good:

```
return array( KernelEvents::CONTROLLER => [ 'getController', 10 ] );
```

I'm not saying you must change it :-)